client/app/components/conference_room/texts/help.md

```diff
@@ -0,0 +1,13 @@
+This is the tablet for our awesome Confy app!
+
+### What is Confy?
+It's the app that makes creating conference room reservations easier than ever before! 
+
+Check it out [here](https://u2i-confy.herokuapp.com).
+
+### What is this tablet for?
+You can view the event that should currently be taking place in this room, as well as the next event if it exists.
```

Comment:
I don't know... but `if it exists` sound very technical... How about: `if there is one` ?